utils/docker/Dockerfile.centos.7

```diff
@@ -1,64 +1,27 @@
 # All rights reserved.
 #
-# Redistribution and use in source and binary forms, with or without
-# modification, are permitted for any purpose (including commercial purposes)
-# provided that the following conditions are met:
-#
-# 1. Redistributions of source code must retain the above copyright notice,
-#    this list of conditions, and the following disclaimer.
-#
-# 2. Redistributions in binary form must reproduce the above copyright notice,
-#    this list of conditions, and the following disclaimer in the
-#    documentation and/or materials provided with the distribution.
-#
-# 3. In addition, redistributions of modified forms of the source or binary
-#    code must carry prominent notices stating that the original code was
-#    changed and the date of the change.
-#
-#  4. All publications or advertising materials mentioning features or use of
-#     this software are asked, but not required, to acknowledge that it was
-#     developed by Intel Corporation and credit the contributors.
-#
-# 5. Neither the name of Intel Corporation, nor the name of any Contributor
-#    may be used to endorse or promote products derived from this software
-#    without specific prior written permission.
-#
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER BE LIABLE FOR ANY
-# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
-# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
-# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
-# THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-#
 # 'recipe' for Docker to build an image of centOS-based
 # environment for building the DAOS project.
 #
 
 # Pull base image
-FROM centos:7.8.2003
+FROM centos:centos7.9.2009
 LABEL maintainer="daos@daos.groups.io"
 
-# Build arguments can be set via --build-arg
-# use same UID as host and default value of 1000 if not specified
-ARG UID=1000
-
 # Install OS updates and package.  Include basic tools and daos dependencies
 # that come from the core repo.
 # Clean up any repos afterwards to save space.
-RUN yum clean all && yum -y install epel-release &&			\
+RUN yum clean all &&							\
+    yum -y install epel-release &&					\
     yum -y upgrade &&							\
     yum -y install							\
 	boost-devel clang-analyzer cmake CUnit-devel doxygen file flex	\
 	gcc gcc-c++ git golang graphviz lcov				\
 	libaio-devel libcmocka-devel libevent-devel libiscsi-devel	\
 	libtool libtool-ltdl-devel libuuid-devel libyaml-devel		\
-	make meson nasm ninja-build numactl-devel openssl-devel		\
```

Comment:
FWIW: These are going to have to get put back when we update spdk/dpdk next time.

Reply:
Our dependencies are all built using this code so I'm not sure I follow why they'll need to be added back, unless we're using this dockerfile to build stuff from a different repo somewhere?

meson, nasn and ninja were all used for Fuse which we don't build any more.

Reply:
Understood.  But meson and ninja will be coming back when we upgrade (not now, but in the future) spdk and dpdk since the latter switched their build system to using meson an ninja.

Was really just making a point of interest.  Nothing needed here yet.